scripts/swagger_auth.py

```diff
@@ -174,13 +175,16 @@ def main(argv=sys.argv[1:]):
                         help='Change the swagger file to include auth on all endpoints.')
     parser.add_argument('-t', '--travis', dest="travis", action='store_true', default=False,
                         help='Run on travis to check that swagger has default auth.')
+    parser.add_argument('-n', '--auth-name', dest="auth_name", default="dcpAuth",
```

Comment:
can we set this default to `OauthSecurity` ? its what is currently being used in the yml.